demo/full/scripts/components/Options/BufferOptions.tsx

```diff
@@ -204,8 +204,10 @@ function BufferOptions({
           {
             maxVideoBufferSize === Infinity ||
             !isMaxVideoBufferSizeLimited ?
-              "Not setting a size limit to the video buffer (relying only on the wantedBufferAhead option)" :
-              `Buffering at most around ${maxVideoBufferSize} kilobyte(s) on the video buffer`
+              `Not setting a size limit to the video buffer (relying only on the 
+                wantedBufferAhead option)` :
```

Comment:
Because backquote keep line breaks, I think this will result in a line break in the UI, which I suppose we don't want